OneCommunityGlobal/HighestGoodNetworkApp
# Nanguan Add test for CreateNewBadgePopup.jsx

# Description
Add unit test for CreateNewBadgePopup.jsx

## Main changes explained:
Added a new file under reducers/tests called CreateNewBadgePopup.test.js
Test the following scenarios:
1. Rendering the component without crashing.
2. Updating the state when the badge name input changes.
3. Updating the state when the image URL input changes.
4. Updating the state when the description input changes.
5. Updating the state when the ranking input changes.
6. Updating the state when the badge type select changes.

## How to test:
1. Checkout the current branch
2. Run npm test CreateNewBadgePopup.test.js
3. Verify all test cases pass without any errors

## Screenshots or videos of changes:
![Screenshot_20241123_230009](https://github.com/user-attachments/assets/8bbb4c70-65c2-4d9f-ade9-3775684af4bf)
